inc/tainetworkif.h

```diff
@@ -511,6 +511,30 @@ typedef enum _tai_network_interface_attr_t
      */
     TAI_NETWORK_INTERFACE_ATTR_NOTIFY,
 
+    /**
+     * @brief The current post-VOA RX input power in the provisioned channel in dBm
+     *
+     * @type #tai_float_t
+     * @flags READ_ONLY
+     */
+    TAI_NETWORK_INTERFACE_ATTR_CURRENT_POST_VOA_PROVISIONED_CHANNEL_POWER,
```

Comment:
@samisaab 
What is the register name for ACO?
I only know "Current Post-VOA Total Power".

Reply:
I'm currently using the four RX power monitoring attributes as follows.

TAI_NETWORK_INTERFACE_ATTR_CURRENT_INPUT_POWER
    Pre-VOA Total Power
    OIF register 0xB4E0: Current Input Power [Total Rx Optical]

TAI_NETWORK_INTERFACE_ATTR_CURRENT_POST_VOA_TOTAL_POWER
    Post-VOA Total Power
    OIF register 0xBBF8: Current Post-VOA Total Power
    We currently use a vendor defined 0x9000 register

TAI_NETWORK_INTERFACE_ATTR_CURRENT_PROVISIONED_CHANNEL_POWER
    Pre-VOA Channel/Signal Power
    OIF register 0xBBF4: Current Provisioned Channel Power

TAI_NETWORK_INTERFACE_ATTR_CURRENT_POST_VOA_PROVISIONED_CHANNEL_POWER
    Post-VOA Channel/Signal Power
    We currently use a vendor defined 0x9000 register

Reply:
Thank you for your information. I see. Then this is good for me.
We are better to confirm by using this kind of information to understand if possible.

Reply:
@samisaab Can you update the description of all of these input power attributes as you commented?

We've also started putting a reference register if it exists.
https://github.com/Telecominfraproject/oopt-tai/blob/master/inc/tainetworkif.h#L470